From 7f809fdcb9b959f90c38ac6388f3e99c0f52c194 Mon Sep 17 00:00:00 2001
From: Hardik Shah <hardik.t.shah@intel.com>
Date: Fri, 29 Apr 2016 14:59:48 +0530
Subject: [PATCH 0153/1061] SDW: Change log level to error from debug.

For any errors, log level should be error.
By mistake it was debug.

Change-Id: I81d55a2efaae172329515f6e000bbed3063400ef
Signed-off-by: Hardik Shah <hardik.t.shah@intel.com>
---
 drivers/sdw/sdw.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/sdw/sdw.c b/drivers/sdw/sdw.c
index 849038266cd3..9e13bfe3579a 100644
--- a/drivers/sdw/sdw.c
+++ b/drivers/sdw/sdw.c
@@ -1838,7 +1838,7 @@ int sdw_config_stream(struct sdw_master *mstr,
 		}
 	}
 	if (!sdw_rt) {
-		dev_dbg(&mstr->dev, "Valid stream tag not found\n");
+		dev_err(&mstr->dev, "Valid stream tag not found\n");
 		ret = -EINVAL;
 		goto out;
 	}
-- 
2.17.1

